### ISSUE TITLE
Enable Travis CI

### ISSUE THREAD

**Issue by njh** [bbc/RadioVisDemo#6]
Something like this should work:

```
sudo: required

language: python

python:
  - "2.6"
  - "2.7"

install:
  - sudo apt-get update -qq
  - sudo apt-get install -y python-wxgtk3.0 python-dnspython python-nose

script:
  - export PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
  - nosetests
```

My only uncertainty is how the different versions of Python managed by Travis work with the Debian packages.

**Comment by njh** [comment]
Implemented in #13

I decided to install the dependencies using `pip` rather than apt, so that it is possible to support multiple different versions of python. Unfortunately it is very slow to install.

Note that Python 2.6 doesn't seem to be available for xenial

**Comment by njh** [comment]
Working: https://travis-ci.org/bbc/RadioVisDemo
